src/llmcompressor/modifiers/factory.py

```
@@ -34,30 +34,64 @@ def refresh():
+
+                yield name, is_pkg
+
+                # If it's a package, recursively walk its submodules
+                if is_pkg:
+                    try:
+                        subpackage = importlib.import_module(name)
+                        if hasattr(subpackage, "__path__"):
+                            yield from walk_recursive(subpackage.__path__, name + ".")
+                    except Exception:
+                        # If we can't import the subpackage, skip it
+                        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)